README.md

```diff
@@ -57,7 +57,7 @@ docker build . --tag veecle/flash-tricore --build-arg=AGREE_INFINEON_TERMS=1 -f
 
 Install `tricore-probe`:
 ```shell
-cargo install tricore-probe --git https://github.com/veecle/tricore-probe --version 0.2.0
+cargo install --path .
```

Comment:
Please revert this change, this is intended for people installing the latest released version.